mesonbuild/backend/ninjabackend.py

```diff
@@ -2013,7 +2013,8 @@ def get_custom_target_dir_include_args(self, target, compiler):
             # own target build dir.
             if not isinstance(i, (build.CustomTarget, build.CustomTargetIndex)):
                 continue
-            idir = self.get_target_dir(i)
+            # NOTE: if we want to use os.path.realpath() for include path only! CK
```

Comment:
I don't understand what this comment even means, (I do know what `os.path.realpath` does). Also, please don't sign your comments, that's what `git blame` is for.

Reply:
done